utils/tables/srgmap_flat_template.html

```diff
@@ -0,0 +1,74 @@
+{{% set title = "SRGs from General Purpose Operating System Security Requirements Guide Mapped to DISA STIG for " ~ full_name -%}}
+
+{{% extends "tables_template_common.html" %}}
+
+{{% block table -%}}
+<table>
```

Comment:
This is almost exactly what `utils/create_srg_export.py` produces as a CSV. Is it worth trying to combine these outputs?

Reply:
Can you explain this more?

Reply:
So the flat SRG table looks very similar to the output of `utils/create_srg_export.py`. They have the same columns, however `utils/create_srg_export.py` does include more statuses and includes more fields that are always blank in the srg mapping tables.

Reply:
Great idea. My problem is that I actually don't understand the tables. So I only tried to create a tool that generates the same tables as the tables that we generate currently by XSLTs. But the current tables are confusing. Why are there empty columns? Why are columns with same values on each row? Why the headers don't correspond the content? I wonder if anyone has answers to this.

Reply:
So the table columns are from the STIG spreadsheet. 

> Why are there empty columns?

Depending on the status column, some columns might be blank. For example if the status is "Applicable - Configurable", the last three columns do not need to be filled out.

> Why are columns with same values on each row?

If there is a CaC rule for a given SRGID it almost always "Applicable - Configurable".

> Why the headers don't correspond the content?

Not sure on this one.

Reply:
What about merging the PR if it works, and do the unification of the codebase with `utils/create_srg_export.py` as a second step? The PR is quite big already.